content/developers/api-reference/scitt-api/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "SCITT API (Preview)"
+description: "SCITT API Reference (Preview)"
+lead: "SCITT API Reference (Preview)"
+date: 2021-06-09T13:49:35+01:00
+lastmod: 2021-06-09T13:49:35+01:00
+draft: false
+images: []
+menu: 
+  developers:
+    parent: "api-reference"
+weight: 112
+toc: true
+aliases: 
+  - /docs/api-reference/scitt-api/
+---
+
+{{< note >}}
+The SCITT API is currently in preview and subject to change.
+{{< /note >}}
+
+The **S**upply **C**hain **I**ntegrity, **T**ransparency and **T**rust (SCITT) initiative is a set of [IETF standards](https://datatracker.ietf.org/group/scitt/documents/) for managing the compliance and auditability of goods and services across end-to-end supply chains.
+SCITT supports the ongoing verification of goods and services where the authenticity of entities, evidence, policy, and artifacts can be assured and the actions of entities can be guaranteed to be authorized, non-repudiable, immutable, and auditable.
+
+DataTrails provides a SCITT implementation for developers looking to integrate Auditing and Compliance capabilities into their services.
+
+## Dependencies
+
+The following are required to complete this walkthrough:
+
+- [A DataTrails subscription](https://app.datatrails.ai/signup)
+- [DataTrails scripts](#data-trails-scripts)
+- [SBOM Tool](https://github.com/microsoft/sbom-tool)
+- [Sample Code](#sample-code)
+
+### Data Trails Scripts
+
+TODO: Add details for installing the datatrails scripts
+
+### Sample Code
+
+Create a directory and clone the [SCITT Examples](https://github.com/scitt-community/scitt-examples) repository
+
+```bash
+mkdir datatrails-scitt-demo
+cd datatrails-scitt-demo/
+git clone https://github.com/scitt-community/scitt-examples.git
+cd scitt-examples/nodejs/
+```
+
+## Sample
+
+The following will accomplish the following tasks:
+
+1. Create a signing key
+1. Register provenance for a new artifact
+1. Create and register an SBOM for the artifact
+1. Create an register attestation for the artifact
+1. Query DataTrails for the statements about the artifact
+
+## Create a Signing Key
+
+DataTrails supports multiple signing keys.
+(TODO: link to supported DataTrails Signing Keys\)<br>
+For a quick start, we'll create a jwk which DataTrails will cryptographically validate upon registration.
+
+1. Create a local signing key
+
+    ```shell
+    datatrails key create <parameters>
```

Comment:
We do not currently have any plans to create a publically available CLI  for creating signed statements or ephemeral keys.

I find a simple coding example much easier than shell commands for this kind or pre-step. The python script is a good example (if tidied up) and we can also do other languages if required.

Reply:
I've updated the sample to use python scripts, rather than an implied CLI.

Reply:
@honourfish fair comment. I like the idea that I can follow the examples with 'no brains required' until I see the value. And that all the help we can provide to get down the happy path. But yes, once I'm there I would want to see code. Re the implied cli, yep I think we just want to be careful to delineate product vs demo and be super clear which we are doing at any point

Reply:
excellent @SteveLasker! and thanks @robinbryce, I agree, and Steve made the same point, a first touch requires runable examples without thinking. I was pondering this, and it feels like the creation of a signed statement is not overly intuitive. 

I quite like Steve's suggestion that we move it over to developer patterns, and we should consider how we can make this easier for a user. For example, make a sample as part of a samples github repo. Thoughts?